src/code/z_actor.c

```diff
@@ -4445,39 +4445,42 @@ s32 func_800BE63C(struct EnBox* chest) {
     return false;
 }
 
-TexturePtr D_801AEFA8[] = {
-    gameplay_keep_Tex_091DE0,
-    gameplay_keep_Tex_091FE0,
-    gameplay_keep_Tex_0921E0,
-    gameplay_keep_Tex_0923E0,
+TexturePtr sElectricShockTextures[] = {
+    gElectricShock1Tex,
+    gElectricShock2Tex,
+    gElectricShock3Tex,
+    gElectricShock4Tex,
 };
```

Comment:
I don't really like this name, but I can't think of a good way to be consistent here.

Reply:
In what sense? 

Do you mean the simple `1/2/3/4` notation used? I wasn't able to find a good way to differentiate the sparks as they alternative between the 4 every consecutive frame, and all 4 textures look very similar.

Do you mean the name `ElectricShock`? Is that too general and could there be a more specific name for this instance?

Do you mean `sElectricShockTextures`, the name that contains all the textures?

Or do you mean something else? For reference:

![image](https://user-images.githubusercontent.com/47598039/153155167-24612ef7-f781-4286-bd91-55b346d17253.png)

Reply:
Ah, yeah, `Sparks` might be a good description. I'm sure there's no good way to distinguish them more than numbers... but should they be 0-indexed?

Reply:
`ElectricShock` -> `ElectricSpark` is good.

![image](https://user-images.githubusercontent.com/47598039/153290372-c033f142-6497-4c98-aef8-207fccb38cb1.png)
These are the 4 spark textures, let me know if you any ideas on names to distinguish these haha.

In regards to the 0-index vs 1-index, take a look at OoT's Gameplay_Keep. There are a lot of textures that are numbers like these are and they are all 1-index. I'm not the biggest fan of 1-index either, but in this case I think it should be made consistent with OoT's Gameplay Keep (things like dust texture names from index 1-8 were copied from OoT so it will impact MM as well).

That might be something worth bringing up in OoT if we want to re-index that

Reply:
Fair enough. And yeah, I'm not engaging in some sort of spark texture Rorschach test lol.